manifests/Rust/rustup/1.21.1.yaml

```diff
@@ -0,0 +1,19 @@
+Id: Rust.rustup
+Publisher: Rust
+Name: rustup
+Version: 1.21.1
+AppMoniker: rustup
+Description: Manage multiple rust installations with ease
+Tags: rust, rustlang, rustup, multirust, install, proxy, toolchain
+Homepage: https://github.com/rust-lang/rustup
+MinOSVersion: 10.0.0.0
+License: MIT or Apache-2.0
+LicenseUrl: https://opensource.org/licenses/MIT or https://opensource.org/licenses/Apache-2.0
+Installers:
+  - Arch: x64
+    Url: https://static.rust-lang.org/rustup/archive/1.21.1/x86_64-pc-windows-gnu/rustup-init.exe
```

Comment:
@rylev I'm not sure if there's a way around rust-lang/rustup#1568. From what I gather, `rustup-init.exe` isn't code signed, triggers a warning and fails to install.

```powershell
cd winget-pkgs\manifests\Rust\rustup\
winget install --manifest 1.21.1.yaml
```
![image](https://user-images.githubusercontent.com/12190184/83106092-27c8fa80-a0bc-11ea-90e7-becd62ef4e58.png)

#123 seems to have been a similar issue.